src/aqueduct-helm/Chart.yaml

```diff
@@ -22,4 +22,3 @@ version: 0.0.16
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "0.0.16"
-namespace: aqueduct
```

Comment:
This doesn't seem to have any effect, so I'm dropping this. From the documentation it seems that we can't specify namespace in `Chart.yaml`, instead we have to use `--namespace` when calling `helm install`.